collector/zfs.go

```diff
@@ -53,22 +43,44 @@ func init() {
 }
 
 type zfsCollector struct {
-	zfsMetrics []zfsMetric
+	zfsMetrics        []zfsMetric
+	linuxProcpathBase string
+	linuxPathMap      map[string]string
 }
 
 func NewZFSCollector() (Collector, error) {
-	return &zfsCollector{}, nil
+	var z zfsCollector
+	z.linuxProcpathBase = "spl/kstat/zfs"
+	z.linuxPathMap = map[string]string{
+		"zfsArc":       "arcstats",
```

Comment:
I don't know if it's too late, but it would be kinda nice to have these metrics have more standard names like `node_zfs_arc` instead of `node_zfsArc`.

This is just my opinion though, so I'll leave it up to a maintainer for the final decision.